components/layout_2020/flow/inline.rs

```diff
@@ -1310,13 +1314,16 @@ impl<'a, 'b> InlineFormattingContextState<'a, 'b> {
         block_sizes_of_content: &LineBlockSizes,
         inline_size: Length,
         is_collapsible_whitespace: bool,
+        is_wrappable_whitespace: bool,
```

Comment:
Now that there are two booleans perhaps this could become a bitflags which should make the code more readable? I'm imagining something like this:

```rust

bitflags! {
    pub struct SegmentContentFlags: u8 {
        const COLLAPSIBLE_WHITESPACE = 0b00000001;
        const WRAPPABLE_WHITESPACE = 0b00000010;
    }
}
```

Reply:
Done!